# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-quartz-private.h
# Commit: 698ee02c6416
# Full Hash: 698ee02c64164310cc03aa69d259960b12cf751f
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-04-29 21:42:31
# Regressor Bug: 739096
# File Overlap Count: 1
# Description:
#   Bug 739096 - Remove support for CGLayer-backed cairo quartz surface. r=jrmuizel
#   
#   This was a performance optimization that we no longer care about.
#   Removing it will avoid the need to merge the additions into the new
#   cairo-quartz-surface code from upstream.
# ==============================================================================

diff --git a/gfx/cairo/cairo/src/cairo-quartz-private.h b/gfx/cairo/cairo/src/cairo-quartz-private.h
--- a/gfx/cairo/cairo/src/cairo-quartz-private.h
+++ b/gfx/cairo/cairo/src/cairo-quartz-private.h
@@ -66,21 +66,16 @@ typedef struct cairo_quartz_surface {
 
     /**
      * If non-null, this is a CGImage representing the contents of the surface.
      * We clear this out before any painting into the surface, so that we
      * don't force a copy to be created.
      */
     CGImageRef bitmapContextImage;
 
-    /**
-     * If non-null, this is the CGLayer for the surface.
-     */
-    CGLayerRef cgLayer;
-
     cairo_rectangle_int_t extents;
 
     cairo_bool_t ownsData;
 } cairo_quartz_surface_t;
 
 typedef struct cairo_quartz_image_surface {
     cairo_surface_t base;
 